addons/website/static/src/builder/plugins/highlight/highlight_plugin.js

```diff
@@ -169,9 +177,21 @@ registry.category("website-plugins").add(HighlightPlugin.id, HighlightPlugin);
 formatsSpecs.highlight = {
     isFormatted: (node) => closestElement(node)?.classList.contains("o_text_highlight"),
     hasStyle: (node) => closestElement(node)?.classList.contains("o_text_highlight"),
-    addStyle: (node, { highlightId }) => {
+    addStyle: (node, { highlightId, thicknessToRestore, colorToRestore }) => {
         node.dispatchEvent(new Event("text_highlight_added", { bubbles: true }));
         node.classList.add("o_text_highlight", `o_text_highlight_${highlightId}`);
+        if (colorToRestore) {
+            node.style.setProperty("--text-highlight-color", colorToRestore);
+        }
+        if (thicknessToRestore) {
+            node.style.setProperty("--text-highlight-width", thicknessToRestore);
+        } else {
+            const style = getComputedStyle(node);
+            node.style.setProperty(
+                "--text-highlight-width",
+                Math.round(parseFloat(style.fontSize) * 0.1) + "px"
+            );
```

Comment:
What is this case? Is'nt it already handled by `makeHighlightSvgs` ?

Reply:
makeHighlightSvgs is called by the mutation observer which is done after the normalize. The plugin needs to put the default highlight width before the added step so the normalization can be done

Reply:
I didn't removed the default in the highlight utils to ensure the retro compatibility by highlight that has been done before the refactoring